libraries/README.md

```diff
@@ -0,0 +1,140 @@
+### Component QA status
```

Comment:
This whole file is less relevant, if I understand correctly. I guess it will go away eventually. Is there reason to edit it down a lot now?

Reply:
I don't feel the need to edit it down for this PR. It's a good historical reference and still relevant to the Libraries themselves.